vault/resource_okta_auth_backend.go

```diff
@@ -291,7 +291,12 @@ func oktaAuthBackendDelete(d *schema.ResourceData, meta interface{}) error {
 }
 
 func oktaAuthBackendExists(d *schema.ResourceData, meta interface{}) (bool, error) {
```

Comment:
The `*Exists()` functions are deprecated so we should be able to remove this function along with the reference here: https://github.com/hashicorp/terraform-provider-vault/blob/9a7fdcfc52fe0e242df10907bdf8e61111adabc6/vault/resource_okta_auth_backend.go#LL31C1-L31C33

Reply:
@benashz I have adjusted the PR and tested the provider locally confirming that this also fixes the issue.